src/test/java/org/folio/circulationbff/controller/CirculationBffControllerTest.java

```diff
@@ -28,15 +34,76 @@
 @ExtendWith(MockitoExtension.class)
 class CirculationBffControllerTest {
 
+  private static final String EXTERNAL_SYSTEM_ID = "externalSystemId";
+  private static final String EXTERNAL_USER_ID = "externalUserId";
+  private static final String TENANT_ID = "tenantId";
   @Mock
   private SearchService searchService;
 
   @Mock
   private MediatedRequestsService mediatedRequestsService;
 
+  @Mock
+  private UserService userService;
+
   @InjectMocks
   private CirculationBffController controller;
 
+  @Test
+  void getExternalUserTestShouldReturnResponseWithBodyAndOkStatus() {
+    User expected = new User();
+    expected.setExternalSystemId(EXTERNAL_SYSTEM_ID);
+    UserCollection userCollection = new UserCollection();
+    userCollection.setUsers(List.of(expected));
+    when(userService.getExternalUser(anyString(), anyString())).thenReturn(userCollection);
+
+    ResponseEntity<User> actual = controller.getExternalUser(EXTERNAL_USER_ID, TENANT_ID);
```

Comment:
If we change `EXTERNAL_USER_ID` and `TENANT_ID` to empty strings or anything else, will the test fail?

Reply:
Changed tests due to changed response logic